### PR TITLE
being used Parachute

### PR DESCRIPTION
**Pull request by evanillaa**

### CODE REVIEW

**Review by phumberdroz — APPROVED** [review]
Looks good to me. This MR adds a restrictions on when you can pull weapons in this case it is not possible to pull when falling or in parachute.

I could imagine this having some performance impact as it runs every 250 ms.

File: client/weapdraw.lua [hunk 1]
```diff
@@ -102,7 +102,7 @@ end)
 Citizen.CreateThread(function()
 	while true do
 		local ped = PlayerPedId()
-		if DoesEntityExist(ped) and not IsEntityDead(ped) then
+		if DoesEntityExist(ped) and not IsEntityDead(ped) and not IsPedInParachuteFreeFall(player) and not IsPedFalling(player) and (GetPedParachuteState(player) == -1 or GetPedParachuteState(player) == 0) then
```
Comment (DmACKGL) [inline client/weapdraw.lua:105]: "player" needs to be changed to "ped"

**Review by DmACKGL — CHANGES_REQUESTED** [review]
Changes needed

**Comment by evanillaa** [comment]
@DmACKGL DONE THANK YOU

**Pull request closed without merging**